packages/core/CalendarManager.ts

```diff
@@ -87,6 +78,10 @@ export const getConnectedCalendars = async (
             },
           };
         }
+        if (destinationCalendar) {
+          destinationCalendar.primaryEmail = primary.email;
+          destinationCalendar.integrationTitle = integration.title;
+        }
```

Comment:
Removed extra check by placing this check after !primary check, thus satisfying the condition with one less check. Minor improvement